### PR TITLE
There is a problem with the configuration.

### PR DESCRIPTION
**Pull request by fro9-dev**

### CODE REVIEW

File: src/init.js [hunk 1]
```diff
@@ -26,11 +26,11 @@ const gitInfo = () => {
  */
 const writePackage = async (template, { user, email }) => {
 	let pkg = {}
-	const name = baseName(process.cwd())
-	const source = `src/${name}.${template}`
+	let name = baseName(process.cwd())
```
Comment (fro9-dev) [inline src/init.js:29]: Is there a better way to keep const?

**Comment by osdevisnot** [comment]
👋 @fro9-dev thank you for the catch.

File: src/init.js (hunk 1 above)
Comment (osdevisnot) [reply src/init.js:29]: how about split on path.delimiter? 

https://nodejs.org/api/path.html#path_path_delimiter

Comment (fro9-dev) [reply src/init.js:29]: Is it possible to process "/" and "\\" separately?

And I think we need to resolve the overwrite of the name in pkg.name.

Comment (osdevisnot) [reply src/init.js:29]: The overwrite is already handled in `merge` right after. 

The path separator is platform specific if you use path.delimiter linked above.

Comment (fro9-dev) [reply src/init.js:29]: Even after'merge'
You need to fix the part where the `name` value is kept as the system folder name.

Comment (osdevisnot) [reply src/init.js:29]: I am not sure I am following here. The code first figures out the name of package based on last part of current working directory. If you had a `package.json` in current working directory, the `merge` will pick up the name from that `package.json`.

Comment (osdevisnot) [reply src/init.js:29]: In Retrospect, this MR can just be this

```
diff --git a/src/init.js b/src/init.js
index 1d1945f..17cdf50 100644
--- a/src/init.js
+++ b/src/init.js
@@ -1,3 +1,4 @@
+import path from 'path'
 import merge from 'deepmerge'
 import sort from 'sort-package-json'
 import cli from '../package.json'
@@ -26,7 +27,7 @@ const gitInfo = () => {
  */
 const writePackage = async (template, { user, email }) => {
 	let pkg = {}
-	const name = process.cwd().split('/').pop()
+	const name = process.cwd().split(path.delimiter).pop()
 	const source = `src/${name}.${template}`
 
 	if (exists('./package.json')) {
```

Comment (fro9-dev) [reply src/init.js:29]: It is set as `const name` in all the codes below.
However, if `pkg.name` exists
`name` must be replaced with `pkg.name`

Is my opinion correct?

Comment (fro9-dev) [reply src/init.js:29]:
```javascript
	const name = process.cwd().split('/').pop()
	const source = `src/${name}.${template}`

	if (exists('./package.json')) {
		pkg = JSON.parse(await read('./package.json'))
	}

	pkg = merge(
		{
			name,
			version: '0.0.0',
			license: 'MIT',
			description: name,
			type: 'module',
			exports: {
				'.': {
					browser: `./dist/${name}.js`,
					import: `./dist/${name}.js`,
					require: `./dist/${name}.cjs.js`,
					umd: `./dist/${name}.umd.js`,
				},
				'./': './',
				'./package.json': './package.json',
			},
			keywords: [name],
		},
		pkg
	)
```
yes pkg merged.
But
`name` is replaced with
`const name = process.cwd().split('/').pop()`

Is this what you intended?

Comment (osdevisnot) [reply src/init.js:29]: oh my bad, I did not see the subsequent usage.

Comment (osdevisnot) [reply src/init.js:29]: and now that I see it, I think there are more cases where this will break.

**Comment by osdevisnot** [comment]
@fro9-dev can you try the latest version 6.1.1 on windows and see if you still see an issue.

File: src/init.js (hunk 1 above)
Comment (fro9-dev) [reply src/init.js:29]: I'll go to your house and try!

thx!

**Comment by fro9-dev** [comment]
All my problems have been solved.

I'll close this PR.

Thank you for your quick feedback 👋

**Pull request closed without merging**